packages/markers/src/common/problem.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { IMarker } from './marker';
+import { Diagnostic } from "vscode-languageserver-types";
+
+export const PROBLEM_KIND = 'problem';
+
+export interface IProblemMarker extends IMarker<Diagnostic> {
```

Comment:
The same here about renaming. Why?

Reply:
Done, use ProblemMarker and not IProblemMarker